### ISSUE TITLE
AJAX lightbox links not working... maybe?

### ISSUE THREAD

**Issue by halfempty** [everythingtype/corrado#17]
These appear in the Advisors section, and on the Contact page. They're working on my local dev site, but not on the WPE production site.

**Comment by halfempty** [comment]
This issue seems to come into play when users are not logged-in to WP.

**Comment by halfempty** [comment]
@gehalber Well, actually... I don't know what's going on here. It's working for me now, even when logged-out. 

Is this issue still coming up for you?

**Comment by gehalber** [comment]
No, actually its not working for me. Sometimes it is and sometimes it isn't. When it does work I don't see the image, unless I go to the permalink page.

I'm on chrome.

**Comment by halfempty** [comment]
@gehalber Hmm, this is a puzzle.

I built the lightboxes to use AJAX, and in such a way that it's fairly straightforward to add them anywhere on the site. This is opposed to Heckscher, where they have to be custom programmed each time. 

I'm hoping to create a reusable component. But it seems to be buggy.

I'm able to toggle the lightbox behavior on and off with a setting. So I've turned it off for now, to stabilize the site while I track down the issue.
